eclass/dist-kernel-utils.eclass

```diff
@@ -72,7 +72,11 @@ dist-kernel_get_image_path() {
 			echo arch/x86/boot/bzImage
 			;;
 		arm64)
-			echo arch/arm64/boot/Image.gz
+			if [[ ${KERNEL_IUSE_SECUREBOOT} ]] && use secureboot; then
+				echo arch/arm64/boot/vmlinuz.efi
+			else
+				echo arch/arm64/boot/Image.gz
+			fi
```

Comment:
While you're touching this, could you merge it into `arm64|riscv` and use `${arch}` below?

Reply:
Sure, Fixed :+1: